core/src/main/java/feign/RequestTemplate.java

```diff
@@ -593,7 +609,7 @@ public String toString() {
    * Replaces query values which are templated with corresponding values from the {@code unencoded}
    * map. Any unresolved queries are removed.
    */
-  public void replaceQueryValues(Map<String, ?> unencoded) {
+  public void replaceQueryValues(Map<String, ?> unencoded, Map<String, Boolean> alreadyEncoded) {
```

Comment:
I see no reason in original method being public either, so I decided to leave overloaded one public too to be consistent.
Should I make new method private instead of overloading public method?